pkg/server/sandbox_status.go

```diff
@@ -93,6 +93,11 @@ func (c *criContainerdService) getIP(sandbox sandboxstore.Sandbox) (string, erro
 		PortMappings: toCNIPortMappings(config.GetPortMappings()),
 	}
 
+	// The network namespace has been closed.
```

Comment:
I think we should move this up to to line 87.

Reply:
Yeah, that's true. :)

Reply:
Done.